src/app/dashboard/dashboard-sensor-details/dashboard-resolve-sensor-details.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@angular/core';
```

Comment:
MB better to add 'resolve' suffix in the end of file name like other resolvers have?